trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
+trello_endpoint='https://api.trello.com/1/'
+
+class item:
+    def __init__(self, id, status, title):
+        self.id = id
+        self.status = status
+        self.title = title
+
+class status:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+# Check if proxy env vars have been set
+try:
+    http_proxy = os.environ['http_proxy']
+except KeyError:
+    pp.pprint('No HTTP proxy found')
+    http_proxy = ''
+try:
+    trello_key = os.environ['TRELLO_KEY']
+    trello_token = os.environ['TRELLO_TOKEN']
```

Comment:
You should update .env.template and the README accordingly.

The README should contain everything the reader needs to know in order to run the app after cloning the repository.